src/View/TemplatePaths.php

```diff
@@ -325,8 +325,13 @@ protected function resolveFilesInFolders(array $folders, $format)
      */
     protected function resolveFilesInFolder($folder, $format)
     {
-        $files = glob($folder . '*.' . $format);
-        return !$files ? [] : $files;
+        if (!is_dir($folder)) {
+            return [];
+        }
+        $directoryIterator = new \RecursiveDirectoryIterator($folder, \FilesystemIterator::FOLLOW_SYMLINKS | \FilesystemIterator::SKIP_DOTS);
```

Comment:
Would it be viable to run Symfony Finder here?

Reply:
Except for Fluid not having any dependencies at all right now, no. I would be happy to reconsider this if we begin adding dependencies, but I'd not do that until we have to. IMHO it doesn't save enough code in this case to justify.

Reply:
jup, that's reasonable